airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -21,11 +21,11 @@
 from typing import TYPE_CHECKING, Any, Optional, Sequence
 
 from jsonpath_ng import parse
-from typing_extensions import Literal
 
 from airflow.exceptions import AirflowException
 from airflow.providers.amazon.aws.hooks.sqs import SqsHook
 from airflow.sensors.base import BaseSensorOperator
+from airflow.typing_compat import Literal
```

Comment:
We'll need to check airflow versions here.

What is the minumum version that the provider now needs, and when did we release a version of Airflow core that contained the "Literal" compat.

Reply:
https://github.com/apache/airflow/pull/16582 - in 2.1.1 is when this was added.

Reply:
Minimum version appears to be 2.1.0 https://github.com/apache/airflow/blob/main/airflow/providers/amazon/provider.yaml#L40

Reply:
Then we can't use it, I am afraid.

Reply:
The eror is clear about it https://github.com/apache/airflow/runs/5191554253?check_suite_focus=true

Reply:
Sounds good, thanks for the feedback. Is there a preferred way I should go about solving this @potiuk ?

Reply:
I am for this solution as well:

```
typing-extensions>=3.7.4;python_version<"3.8" 
```